app/models/story.rb

```diff
@@ -183,7 +184,8 @@ def date_for_boolean(value)
 
   # raising exceptions here to prevent sending publish messages
   #  when not actually a change to be published
-  def publish!(apublished_at = DateTime.now)
+  def publish!(apublished_at = nil)
```

Comment:
do we still need the parameter if it's being set in the next line no matter what's passed in? also, looks like it's never called with a param within CMS: 
![screen shot 2017-01-05 at 11 12 08 am](https://cloud.githubusercontent.com/assets/16007382/21687588/d888cbe4-d337-11e6-94b9-752ce60bfd73.png)